config/samples/druid_v1alpha1_etcd.yaml

```diff
@@ -73,6 +73,6 @@ spec:
           app: etcd-statefulset
 
   replicas: 1
-# priorityClassName: foo
+  # priorityClassName: priority-class-name
```

Comment:
In this PR we have not touched this part at all and remains out of scope of this PR. However, just an observation - we support both a single node etcd cluster and a multi-node etcd cluster. Since this is a sample, we have to choose one value. The consumer can easily change the replicas to be equal to an acceptable odd number.

Reply:
I would suggest as this is a very small change we can do this in this PR.
> Since this is a sample, we have to choose one value. The consumer can easily change the replicas to be equal to an acceptable odd number.

if `spec.replicas` is `3` then it seems obvious that `1` will also be supported but if `spec.replicas` is `1` in sample then it may not be implicit that it supports `3`(>1) . Atleast this is how I see it but I leave it to you to decide.

Reply:
Sure, we can progress the sample etcd to 3 replicas in this PR.